tests/output.rs

```diff
@@ -1180,3 +1191,440 @@ fn test_specify_explicit_file_of_unrecognized_file_type_and_language_flag() {
         "#,
     );
 }
+
+#[test]
+fn test_swift() {
+    assert_sorted_output(
+        "swift_project",
+        r#"
+            $ tree-sitter-grep -q '(value_argument) @c' --language swift
+            example.swift:2:    atPath: "native"
+        "#,
+    );
+}
+
+#[test]
+fn test_swift_auto_language() {
+    assert_sorted_output(
+        "swift_project",
+        r#"
+            $ tree-sitter-grep -q '(value_argument) @c'
+            example.swift:2:    atPath: "native"
+        "#,
+    );
+}
+
+#[test]
+fn test_objective_c() {
+    assert_sorted_output(
+        "objective_c_project",
+        r#"
+            $ tree-sitter-grep -q '(struct_declaration) @c' --language objective-c
+            example.h:4:@property (nonatomic, strong, nullable) NSString *baseURL;
+        "#,
+    );
+}
+
+#[test]
+fn test_objective_c_auto_language() {
+    assert_sorted_output(
+        "objective_c_project",
+        r#"
+            $ tree-sitter-grep -q '(struct_declaration) @c'
+            example.h:4:@property (nonatomic, strong, nullable) NSString *baseURL;
+        "#,
+    );
+}
+
+#[test]
+fn test_objective_c_auto_language_ambiguous_query() {
+    assert_failure_output(
+        "objective_c_project",
+        r#"
+            $ tree-sitter-grep -q '(identifier) @c'
+            File "./example.h" has ambiguous file-type, could be ObjectiveC, C or Cpp. Try passing the --language flag
```

Comment:
I think of "Cpp" as the name you use for C++ when you're in a context where using "+" would be annoying/impossible, but not really as the "real" name of the language. (Although I suppose you'd have to quote "c++" to pass it to `--language`, so maybe being consistent here is best?)